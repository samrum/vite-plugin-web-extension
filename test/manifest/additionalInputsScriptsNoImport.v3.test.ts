import { getResourceDir, runManifestV3Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("additionalInputsScriptsNoImport");

runManifestV3Test("additionalInputsScriptsNoImport", () => ({}), {
  additionalInputs: {
    scripts: [`${resourceDir}/script1.js`, `${resourceDir}/script2.js`],
  },
});
