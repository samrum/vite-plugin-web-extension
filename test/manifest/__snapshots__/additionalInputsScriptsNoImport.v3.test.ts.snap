// Vitest Snapshot v1

exports[`additionalInputsScriptsNoImport - Manifest V3 1`] = `
[
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsNoImport/script1.js",
    "name": undefined,
    "source": "console.log(\\"script1\\");
",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsNoImport/script2.js",
    "name": undefined,
    "source": "console.log(\\"script2\\");
",
    "type": "asset",
  },
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 3,
  \\"web_accessible_resources\\": [
    {
      \\"resources\\": [
        \\"test/manifest/resources/additionalInputsScriptsNoImport/script1.js\\"
      ],
      \\"matches\\": [
        \\"<all_urls>\\"
      ],
      \\"use_dynamic_url\\": true
    },
    {
      \\"resources\\": [
        \\"test/manifest/resources/additionalInputsScriptsNoImport/script2.js\\"
      ],
      \\"matches\\": [
        \\"<all_urls>\\"
      ],
      \\"use_dynamic_url\\": true
    }
  ]
}",
    "type": "asset",
  },
]
`;
